src/Bridge/Symfony/Bundle/Test/Client.php

```diff
@@ -121,7 +122,8 @@ public function request(string $method, string $url, array $options = []): Respo
             'url' => $resolvedUrl,
             'primary_port' => 'http:' === $url['scheme'] ? 80 : 443,
         ];
-        $this->kernelBrowser->request($method, $resolvedUrl, [], [], $server, $options['body'] ?? null);
+        $extra = $options['extra'] ?? [];
+        $this->kernelBrowser->request($method, $resolvedUrl, [], $extra['files'] ?? [], $server, $options['body'] ?? null);
```

Comment:
you can use `$options['extra']['files'] ?? []` directly here

Reply:
I've updated it based on your suggestion